src/test/scala/vsys/blockchain/state/opcdiffs/IfOpcDiffTest.scala

```diff
@@ -75,7 +75,7 @@ class IfOpcDiffTest extends PropSpec with PropertyChecks with GeneratorDrivenPro
       Array[Byte](2.toByte, 0.toByte, 1.toByte, 2.toByte),
       Seq[DataEntry](DataEntry(Array(0.toByte), DataType.Boolean),
         DataEntry(Shorts.toByteArray(0), DataType.OpcBlock),
-        DataEntry(Shorts.toByteArray(1), DataType.OpcBlock))) should be (
+        DataEntry(Array[Byte](0, 2, 0, 1), DataType.OpcBlock))) should be (
```

Comment:
OpcBlock DataEntry like L57 should be invalid since it passes the parameters directly.

Reply:
Do you mean that ` DataEntry(Shorts.toByteArray(0), DataType.OpcBlock)` should be invalid, which stands for empty opc block?

Reply:
right, `Array[Byte](0, 2, 0, 0)` is the correct one. and in previous lines, `DataEntry(Shorts.toByteArray(0), DataType.OpcBlock)` used as valid case but actually it is not the valid case in updated version.

Reply:
What ablout empty opc block `Array[Byte](0, 0)`, which has length of zero?

Reply:
I see. Difference between Seq() and Seq(Seq()).